tests/fixtures/log_lines.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This fixture file doesn't appear to be used

Reply:
@liamkeily Yeah, it's a hang up from when I forked the Symfony branch. I'll remove it.